crates/re_viewer_context/src/lib.rs

```diff
@@ -52,11 +52,33 @@ impl SpaceViewId {
         Self(uuid::Uuid::new_v4())
     }
 
+    pub fn random_from_str(s: &str) -> Self {
+        // TODO(jleibs): This is overkill; we can probably get away with 64-bit ids.
+        use rand::{Rng as _, SeedableRng as _};
+        use std::hash::{Hash as _, Hasher as _};
+
+        let salt: u64 = 0x307b_e149_0a3a_5552;
+
+        let mut hasher = std::collections::hash_map::DefaultHasher::default();
```

Comment:
It's worth noting that this hash can change when we update Rust versions.

We use `ahash::RandomState` in many other places, though that can also change when we update `ahash`. However, new rust versions is outside our control, while ahash updates is something we always opt-in to.

TLDR: using `ahash::RandomState::with_seeds(1, 2, 3, 4)` instead here should be a bit better